### PR TITLE
schemawatch: Exclude views from SchemaData.Order

### PR DESCRIPTION
**Pull request by bobvawter**
The queries used to compute table dependency order does not exclude views. This
change updates the queries and adds a test.

X-Ref: #617

- - -
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cdc-sink/618)

### CODE REVIEW

**Review by BramGruneir — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cdc-sink/618#-:-Nl9YjMX7eiJs-4PqAe9:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @sravotto)

File: internal/target/schemawatch/dependencies.go [hunk 1]
```diff
@@ -165,7 +166,8 @@ const depOrderTemplateCRDB = `
 WITH RECURSIVE
  tables AS (
    SELECT schema_name AS sch, table_name AS tbl
-   FROM [SHOW TABLES FROM %[1]s]),
+   FROM [SHOW TABLES FROM %[1]s]
```
Comment (rafiss) [inline internal/target/schemawatch/dependencies.go:169]: just a drive-by comment, but SHOW statements are not considered stable APIs: https://www.cockroachlabs.com/docs/stable/api-support-policy#apis

so ideally this query would use pg_catalog or information_schema, so that it is less likely to break across different versions of CRDBl

Comment (bobvawter) [reply internal/target/schemawatch/dependencies.go:169]: We can remove this query and use the vanilla-PG one in its place once sufficiently-old CRDB versions are phased out of what we want cdc-sink to support as a target.
